package.json

```diff
@@ -41,7 +41,6 @@
     "ember-cli-moment-shim": "0.6.1",
     "ember-cli-qunit": "1.0.1",
     "ember-cli-uglify": "^1.0.1",
-    "ember-data": "~2.0.0",
```

Comment:
We should probably add a handful of acceptance tests to ensure the dummy app still functions...